Audio list - Populated amplitude_list for video object

ewega opened the pull request:
To test:
From cmd,
`cd audio-analysis`
`python main.py`

Run it using any option (1, 2, or 3)

ewega committed:
Populated amplitude_list in Video Object

ewega committed:
Updated with Master

ewega committed:
Merge branch 'master' into audio-list

ewega committed:
Clean up

ewega committed:
Cleanup part 2

tjlam commented:
We should remove those audio files cus they're really big. I think github has a limit that we'll eventually hit. To remove them from the PR, add their file path to the .gitignore file

tjlam commented on audio-analysis/main.py:
```diff
@@ -11,18 +11,22 @@ def saveVideo(url, tag):
     video.getTextAnalysis()
     video.getAmplitudeAnalysis()
 
+    #For testing, will be removed after PR is approved
```
You can make a constant called `TESTING = false`, Add an `if (TESTING) {` so you don't need to delete it

ewega replied:
True, was just trying to keep it consistent with how @mattlianje did it. I'll add the constant though since it makes more sense

tjlam commented on audio-analysis/videoHelper.py:
```diff
@@ -130,10 +137,16 @@ def getAmplitudeAnalysis(self):
 
-        for x, y in interval_amplitudes.items():
-            print(x, y)
+        #Commented out but decided to keep in case, loop through interval_amplitudes intervals and outputs them
```
Maybe we can add this inside a `TESTING` if statement?

ewega replied:
Agreed - I'm almost tempted to just completely remove it though because the dictionary should output the same things anyways

tjlam commented on audio-analysis/videoHelper.py:
```diff
@@ -107,7 +107,7 @@ def getAmplitudeAnalysis(self):
         raw_amplitudes = {}
 
-        fc, ic = 1, 1
+        fc, ic, fc_continuous = 1, 1, 1
```
What does fc, ic, and fc_continuous mean?

ewega replied:
**fc** = frame count (this is reset after going through every interval)
_note that an interval can be 1, 5, or 10 seconds depending on audio time length_
**fc_continuous** = frame count continuous, same as above but not reset after every interval
**ic** = interval count, self explanatory.

tjlam replied:
Ahh makes sense thanks 👍

tjlam reviewed (changes requested):
Looks good just a few suggestions

ewega committed:
Debugging and Cleanup

Adding TESTING variable on main.py and videoHelper.py

Deleted all those WAV files

Eldrick19 commented:
Made suggested changes. Feel free to drop an approval ;) <3

tjlam reviewed (approved):
YEET 🍆
